src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -38,6 +38,17 @@
     <CoreCompileDependsOn>$(CoreCompileDependsOn);GenerateAssemblyInfo</CoreCompileDependsOn>
   </PropertyGroup>
 
+  <Target Name="DecideIfWeNeedToIncludeDllSafeSearchPathAttribute" 
+    BeforeTargets="GenerateAssemblyInfo">
+    
+    <!-- We want to apply the IncludeDllSafeSearchPathAttribute on all DotNet assemblies (non test) that have a reference to System.Runtime.InteropServices -->
+    <PropertyGroup Condition="'$(IncludeDllSafeSearchPathAttribute)'=='' AND '$(IsDotNetFrameworkProductAssembly)' == 'true' AND '$(IsTestProject)' != 'true'">
```

Comment:
minor: I would probably condition this entire target with `'$(IsDotNetFrameworkProductAssembly)' == 'true' AND '$(IsTestProject)' != 'true'`